libs/rxtx/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```

Comment:
This copyright header is outdated and it is recommended that the original author of the content take the copyright unless they really have a signed contract with OpenWrt.org